### ISSUE TITLE
Improvements to "eye/visibility" icon

### ISSUE THREAD

**Issue by jtomeck** [Slicer/Slicer#6683]
## Is your feature request related to a problem? Please describe.
<img width="45" alt="Screen Shot 2022-11-18 at 10 36 00 AM" src="https://user-images.githubusercontent.com/1560779/202742472-b3f4e00a-1f2b-4cf1-8456-39a77f6f74d2.png">
The eye icon, when not "toggled" is not clear that it is an eye, nor that it is clickable.

<img width="48" alt="Screen Shot 2022-11-18 at 10 36 07 AM" src="https://user-images.githubusercontent.com/1560779/202742471-89c1515b-cb84-4824-a044-d8e02ee17644.png">
Notice when it's clicked it looks like a clickable button

## Describe the solution you'd like

- Keep the button outline whether it's enabled or disabled.
- Keep the same eye icon, but when it's not enabled, put a line through the eye.

**Comment by jamesobutler** [comment]
Yes there are some issues with icons that have dark coloring being difficult to see when using the "Dark Slicer" appearance. Another similar issue is https://github.com/Slicer/Slicer/issues/5114. You can go to Edit->Application Settings->Appearance and switch style to "Light Slicer" which will look like:
![image](https://user-images.githubusercontent.com/15837524/202747006-939eed72-62a1-4d9f-b38b-758286f63cf2.png)
![image](https://user-images.githubusercontent.com/15837524/202747037-67475d37-fb02-47af-b2fc-4ad97efc9e24.png)

We have some other issues open on this topic to improve icons for better visibility in both light and dark modes, but have had little access to designers to create these.

The button outline for QToolButtons are only shown when they are hovered over based on standard Qt behavior. It looks like the following
![image](https://user-images.githubusercontent.com/15837524/202746783-e9d01c6d-5cef-41bb-b8f5-f204794e6d9f.png)
![image](https://user-images.githubusercontent.com/15837524/202747267-de81ef4a-ccf7-4a42-9bd9-fc32b40a02fd.png)
